DataAccess/Infrastructure/Dapper/DbAccesserDapper.cs

```diff
@@ -0,0 +1,89 @@
+﻿using DataAccess.Interfaces;
+using Microsoft.Data.SqlClient;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data;
+using System.Text;
+using Dapper;
+using System.Linq;
+using DataAccess.Attrubutes;
+using System.Linq.Expressions;
+
+namespace DataAccess.Infrastructure.Dapper
+{
+    public class DbAccesserDapper<T> : IDbAccesser<T>
+    {
+        public void AddItem(T item)
+        {
+            InsertItem(item);
+        }
+
+        public void DeleteItem(T item)
+        {
+            throw new NotImplementedException();
```

Comment:
If you see 'throw new NotImplementedException();' and class inherits from an interface, you can say that it is an Interface segregation principle violation.